metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.0.2'
+version           '4.0.3'
```

Comment:
In the last PR @rshade ask me to update version in metadata.
In the next PR, if any, should I do it or not?